### YamlMime:ManagedReference
items:
- uid: System.ContextMarshalException
  id: ContextMarshalException
  children:
  - System.ContextMarshalException.#ctor
  - System.ContextMarshalException.#ctor(System.String)
  - System.ContextMarshalException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.ContextMarshalException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ContextMarshalException
  nameWithType: ContextMarshalException
  fullName: System.ContextMarshalException
  type: Class
  summary: "Unntak oppstod under forsøk på å formidle et objekt i en kontekstgrense mislykkes."
  remarks: "Objekter kan formidle etter verdien eller referanse. Forsøk på å sende en type som ikke kan formidles via en kontekstgrense vil føre til en ContextMarshalException.       ContextMarshalException bruker HRESULT COR_E_CONTEXTMARSHAL som har verdien 0x80131504L.       ContextMarshalException bruker standard <xref:System.Object.Equals%2A>implementering, som støtter referanse likhet.</xref:System.Object.Equals%2A>       En liste med opprinnelige verdier for en forekomst av ContextMarshalException, kan du se den <xref:System.ContextMarshalException.%23ctor%2A>konstruktører.</xref:System.ContextMarshalException.%23ctor%2A>"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class ContextMarshalException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.ContextMarshalException.#ctor
  id: '#ctor'
  parent: System.ContextMarshalException
  langs:
  - csharp
  name: ContextMarshalException()
  nameWithType: ContextMarshalException.ContextMarshalException()
  fullName: System.ContextMarshalException.ContextMarshalException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ContextMarshalException&quot;> </xref> klasse standardegenskapene."
  remarks: "Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.ContextMarshalException>.</xref:System.ContextMarshalException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| En tom streng (&quot;&quot;). |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ContextMarshalException ();
    parameters: []
  overload: System.ContextMarshalException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ContextMarshalException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.ContextMarshalException
  langs:
  - csharp
  name: ContextMarshalException(String)
  nameWithType: ContextMarshalException.ContextMarshalException(String)
  fullName: System.ContextMarshalException.ContextMarshalException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ContextMarshalException&quot;> </xref> klassen med en angitt feilmelding."
  remarks: "Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.ContextMarshalException>.</xref:System.ContextMarshalException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ContextMarshalException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
  overload: System.ContextMarshalException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ContextMarshalException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.ContextMarshalException
  langs:
  - csharp
  name: ContextMarshalException(SerializationInfo,StreamingContext)
  nameWithType: ContextMarshalException.ContextMarshalException(SerializationInfo,StreamingContext)
  fullName: System.ContextMarshalException.ContextMarshalException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ContextMarshalException&quot;> </xref> klasse serialiserte dataene."
  syntax:
    content: protected ContextMarshalException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Den <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> som inneholder den serialiserte objektdata om som oppstod et unntak."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Den <xref:System.Runtime.Serialization.StreamingContext>som inneholder kontekstavhengig informasjon om kilden eller målet.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.ContextMarshalException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ContextMarshalException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.ContextMarshalException
  langs:
  - csharp
  name: ContextMarshalException(String,Exception)
  nameWithType: ContextMarshalException.ContextMarshalException(String,Exception)
  fullName: System.ContextMarshalException.ContextMarshalException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ContextMarshalException&quot;> </xref> klassen med en angitt feilmelding og en referanse til det interne unntaket er årsaken til dette unntaket."
  remarks: "Et unntak som er registrert som en direkte resultatet av et tidligere unntak må inkludere en referanse til det tidligere unntaket i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>egenskapen returnerer den samme verdien som er sendt til konstruktøren, eller `null` hvis den <xref:System.Exception.InnerException%2A>egenskapen leverer ikke internt unntak verdien til konstruktøren.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.ContextMarshalException>.</xref:System.ContextMarshalException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referansen for det indre unntaket. |   | <xref:System.Exception.Message%2A>| Feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ContextMarshalException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
    - id: inner
      type: System.Exception
      description: "Unntaket er årsaken til den gjeldende unntaket. Hvis den `inner` -parameteren er ikke <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, oppstår det gjeldende unntaket i en <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokken som behandler det interne unntaket."
  overload: System.ContextMarshalException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.ContextMarshalException.#ctor
  parent: System.ContextMarshalException
  isExternal: false
  name: ContextMarshalException()
  nameWithType: ContextMarshalException.ContextMarshalException()
  fullName: System.ContextMarshalException.ContextMarshalException()
- uid: System.ContextMarshalException.#ctor(System.String)
  parent: System.ContextMarshalException
  isExternal: false
  name: ContextMarshalException(String)
  nameWithType: ContextMarshalException.ContextMarshalException(String)
  fullName: System.ContextMarshalException.ContextMarshalException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.ContextMarshalException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.ContextMarshalException
  isExternal: false
  name: ContextMarshalException(SerializationInfo,StreamingContext)
  nameWithType: ContextMarshalException.ContextMarshalException(SerializationInfo,StreamingContext)
  fullName: System.ContextMarshalException.ContextMarshalException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.ContextMarshalException.#ctor(System.String,System.Exception)
  parent: System.ContextMarshalException
  isExternal: false
  name: ContextMarshalException(String,Exception)
  nameWithType: ContextMarshalException.ContextMarshalException(String,Exception)
  fullName: System.ContextMarshalException.ContextMarshalException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.ContextMarshalException.#ctor*
  parent: System.ContextMarshalException
  isExternal: false
  name: ContextMarshalException
  nameWithType: ContextMarshalException.ContextMarshalException
